breacharbiter.go

```diff
@@ -544,11 +544,12 @@ justiceTxBroadcast:
 	// With the breach transaction confirmed, we now create the
 	// justice tx which will claim ALL the funds within the
 	// channel.
-	finalTx, err := b.createJusticeTx(breachInfo)
+	justiceTxs, err := b.createJusticeTx(breachInfo)
```

Comment:
Typos in commit message: 
  * splitted -> split (or distinct/piece-wise)? 
  * justie -> justice